source/_integrations/version.markdown

```diff
@@ -15,44 +15,9 @@ ha_platforms:
   - sensor
 ---
 
-The `version` sensor platform that can display the current Home Assistant versions.
+The `version` integration can display the current Home Assistant versions.
```

Comment:
Removing the backticks, as its no longer used in configuration :)

```suggestion
The Version integration can display the current Home Assistant versions.
```

Reply:
`{{ page.title }}` :smile:

Reply:
Sure, keeping stuff normal readable is a good thing as well :)